apps/marketplace/src/components/ArtistSpotlight.tsx

```diff
@@ -0,0 +1,104 @@
+import { FunctionComponent, useEffect, useState } from "react";
+import { Grid, Stack, Typography } from "@mui/material";
+import Artist from "./Artist";
+
+const ArtistSpotlight: FunctionComponent = () => {
+  // TEMP: simulate data loading
+  const [isLoading, setIsLoading] = useState(true);
+
+  const handleSelectArtist = (name: string) => {
+    // placeholder
+    console.log("artist selected: ", name); // eslint-disable-line
+  };
+
+  // TEMP: simulate data loading
+  useEffect(() => {
+    setTimeout(() => {
+      setIsLoading(false);
+    }, 1000);
+  }, []);
+
+  return (
+    <Stack mb={ 4 }>
+      <Stack alignItems="center" mb={ 5 } mt={ 20 }>
+        <Typography variant="h3">ARTIST SPOTLIGHT</Typography>
```

Comment:
```suggestion
           <Typography fontSize={ ["24px", "24px", "32px"] } variant="h3">
            ARTIST SPOTLIGHT
          </Typography>
```
to align with the just released tile